src/app/vault/bulk-actions.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="dropdown mr-2" appListDropdown>
+    <button class="btn btn-sm btn-outline-secondary dropdown-toggle" type="button" id="bulkActionsButton"
+        data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" appA11yTitle="{{'options' | i18n}}">
+        <i class="fa fa-cog" aria-hidden="true"></i>
+    </button>
+    <div class="dropdown-menu dropdown-menu-right" aria-labelledby="bulkActionsButton">
+        <button class="dropdown-item" appStopClick (click)="bulkMove()" *ngIf="!deleted && !organization">
+            <i class="fa fa-fw fa-share" aria-hidden="true"></i>
+            {{'moveSelected' | i18n}}
+        </button>
+        <button class="dropdown-item" appStopClick (click)="bulkShare()" *ngIf="!deleted && !organization">
+            <i class="fa fa-fw fa-share-alt" aria-hidden="true"></i>
+            {{'shareSelected' | i18n}}
+        </button>
+        <button class="dropdown-item" (click)="bulkRestore()" *ngIf="deleted && !organization">
+            <i class="fa fa-fw fa-undo" aria-hidden="true"></i>
+            {{'restoreSelected' | i18n}}
+        </button>
```

Comment:
I am guessing we would probably want restore bulk action too, right @cscharf ? Maybe a fast follow?

Reply:
There is a backlog item for other bulk items, including restore. I imagine I'll grab that as soon as it's planned and our August stuff is all wrapped up.